src/Components/ColorForm/ColorForm.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { nanoid } from "nanoid";
+import ColorInput from "./ColorInput";
+import "./ColorForm.css";
+
+export default function ColorForm({ onAddColor }) {
+  const [role, setRole] = useState("");
+  const [hex, setHex] = useState("#ff0000");
+  const [contrastText, setContrastText] = useState("#ffffff");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
please don't use "e". These kinds of abbreviations are quite common but when reading code its best to have more explicit names. e only makes sense in context, it can be "error" or "event" or something else. Reducing the mental load for your readers of your code is important for its maintainability.

Reply:
thanks, i changed it to event :)